LAKMobile/src/screens/LoginScreen.tsx

```diff
@@ -1,18 +1,36 @@
 import React, { useContext, useState } from 'react';
-import { StyleSheet, View } from 'react-native';
+import { StyleSheet, Text, View } from 'react-native';
 import { AppText, LabelWrapper, AppButton, AppTextInput } from '../components';
 import { COLORS } from '../../constants';
 import { LoginProps } from '../types/navigation';
 import { AuthContext } from '../context/AuthContext';
+import { useEffect } from 'react';
 
 export function LoginScreen({ navigation }: LoginProps) {
   const [phoneNumber, setPhoneNumber] = useState('');
   const [pin, setPIN] = useState('');
 
+  const [isPhoneValid, setIsPhoneValid] = useState(false);
+  const [isPINValid, setIsPINValid] = useState(false);
+
+  const [isLoginPressed, setIsLoginPressed] = useState(false);
```

Comment:
I think that we should name these state variables phoneValid, setPhoneValid, pinValid, setPINValid, loginPressed, setLoginPressed just to keep things concise.